### PR TITLE
Stargate discovery

### PR DESCRIPTION
**Pull request by krzkaczor**
For now, we decided to ignore Pool Factory contract. By doing this we avoid tracking token pools which don't have owner but are storing all the funds. 

@sz-piotr @michalsidzej @luke7211 @antooni thoughts?

### CODE REVIEW

File: packages/backend/discovery/stargate/discovered.json [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+        "getChainId": 1,
+        "getOwners": [
+          "0x9F403140Bc0574D7d36eA472b82DAa1Bbd4eF327",
+          "0xe095F2590eF1Ab39601445025847Ed8E4B40D687",
+          "0xBb6633cc267951E938F9B6421E4F54aa5b2c1936",
+          "0x73E9c017Ad37e2113e709D8070Cc9E1b28180e1e",
+          "0x67FC8c432448f9a8d541C17579EF7a142378d5aD"
+        ],
+        "getThreshold": 2,
+        "nonce": 21,
```
Comment (antooni) [inline packages/backend/discovery/stargate/discovered.json:98]: Probably you should ignore this method in watchmode

File: packages/backend/discovery/stargate/discovered.json [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+      "values": {
+        "BLOCK_VERSION": 65535,
+        "chainId": 1,
+        "DEFAULT_VERSION": 0,
+        "defaultReceiveLibraryAddress": "0x4D73AdB72bC3DD368966edD0f0b2148401A178E2",
+        "defaultReceiveVersion": 2,
+        "defaultSendLibrary": "0x4D73AdB72bC3DD368966edD0f0b2148401A178E2",
+        "defaultSendVersion": 2,
+        "getChainId": 1,
+        "isReceivingPayload": false,
```
Comment (antooni) [inline packages/backend/discovery/stargate/discovered.json:73]: ignore in watchmode?

File: packages/backend/discovery/stargate/discovered.json [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+        "isSendingPayload": false,
```
Comment (antooni) [inline packages/backend/discovery/stargate/discovered.json:74]: ignore in watchmode?

**Review by michalsidzej — APPROVED** [review]
As Pool factory doesn't have an owner it makes sense to skip it

File: packages/backend/discovery/stargate/discovered.json [hunk 4]
```diff
@@ -0,0 +1,437 @@
...
+        "getChainId": 1,
+        "getOwners": [
+          "0x1D7C6783328C145393e84fb47a7f7C548f5Ee28d",
+          "0x285b7EEa81a5B66B62e7276a24c1e0F83F7409c1",
+          "0x79e2b9C1F6C9ed1375C93AaF139e6C4537f48523",
+          "0xF05F4211ad15A8e49b49C0436067CFFfEa783aA4",
+          "0xf02CC4dc84aC59Bd6089BAddcEB9d4Ef3AEFb0f0"
+        ],
+        "getThreshold": 2,
+        "nonce": 300,
```
Comment (antooni) [inline packages/backend/discovery/stargate/discovered.json:39]: ignore in watchmode?

**Review by antooni — APPROVED** [review]

**Pull request merged**